src/libs/getButtonState.js

```diff
@@ -0,0 +1,20 @@
+import CONST from '../CONST';
+
+/**
+ * Get the string representation of a button's state.
+ *
+ * @param {Boolean} [isHovered]
+ * @param {Boolean} [isPressed]
+ * @returns {String}
+ */
+export default function (isHovered = false, isPressed = false) {
```

Comment:
Since this is only used once I'm going to suggest moving it to your component.

Reply:
We _could_, do you not think this is something that would be generically useful?

Reply:
bump